lib/portlayer/storage/vsphere/image.go

```diff
@@ -366,7 +366,7 @@ func (v *ImageStore) scratch(ctx context.Context, storeName string) error {
 	log.Infof("Creating image %s (%s)", portlayer.Scratch.ID, imageDiskDsURI)
 
 	// Create the disk
-	vmdisk, err := v.dm.CreateAndAttach(ctx, imageDiskDsURI, "", defaultDiskSize, os.O_RDWR)
+	vmdisk, err := v.dm.CreateAndAttach(ctx, imageDiskDsURI, "", portlayer.Config.ScratchSize, os.O_RDWR)
```

Comment:
I can do this during `vic-machine create` time @fdawg4l so that we bail & do not create the VCH if 0 is specified, or we can log a warning and supply the default at that time. We could also impose a sane lower limit instead of 0.